src/cljs/ui/splash/page.cljs

```diff
@@ -13,81 +13,156 @@
-                                                                :src     (arg->icon (:spread icons))}])
-                       :title      (reagent/as-element [typography {:class-name (:typography classes)}
-                                                        "Spread"])}]]
+
+        [card-header {:class-name (:card-header classes)
+                      :avatar     (reagent/as-element [avatar {:alt     "spread"
+                                                               :variant "square"
+                                                               :src     (arg->icon (:spread icons))}])
+                      :title      (reagent/as-element [typography {:class-name (:card-header-title classes)}
+                                                       "Spread"])}]
+
         [divider {:variant "fullWidth"}]
-        [card-content {:class-name (:centered classes)}
-         [:div {:class-name (:middle classes)}
-          [button {:class-name (:button classes)
-                   :variant    "contained"
-                   :start-icon (reagent/as-element [:img {:src (arg->icon (:google icons))}])
-                   :href       (str "https://accounts.google.com/o/oauth2/v2/auth"
-                                    "?client_id=" client-id
-                                    "&scope=email%20profile"
-                                    "&response_type=code"
-                                    "&redirect_uri=" (utils/url-encode redirect-uri))}
-           "Continue with Google"]]]]])))
+
+        [card-content {:class-name (:card-content classes)}
+
+         [typography {:class-name (:sign-in classes)}
+          "Sign in with email"]
+
+         [typography {:class-name (:sign-in-sub classes)}
+          "Enter your email address. We will send you a special link that you can sign in with instantly."]
+
+         [text-input {:label       "E-mail address"
+                      :opts        {:style {:width   "325px"
+                                            :margin  "10px"
+                                            :padding "10px"}}
+                      :error?      error
+                      :helper-text (when error
+                                     "Enter valid email address")
+                      :value       email
+                      :on-change   (fn [value]
+                                     (if (email? value)
+                                       (setError false)
+                                       (setError true))
+                                     (setEmail value))}]
+
+         [button {:class-name (:send-button classes)
+                  :variant    "contained"
+                  :disabled   error
+                  :on-click   (fn []
+                                (>evt [:splash/send-login-email email root-url])
+                                (setEmail nil))}
+          "Send magic link"]
+
+         [:h2 {:style {:width         "325px"
```

Comment:
I was also adding all this styles to ui/css/main.css or ui/css/components.css if they are shared components

Reply:
done